toolchain/sem_ir/ids.h

```diff
@@ -710,12 +710,17 @@ constexpr TypeBlockId TypeBlockId::Invalid = TypeBlockId(InvalidIndex);
 // An index for element access, for structs, tuples, and classes.
 struct ElementIndex : public IndexBase, public Printable<ElementIndex> {
   using IndexBase::IndexBase;
+
+  static const ElementIndex Invalid;
```

Comment:
As noted on my other comment, the existing code that handles `ElementIndex` values isn't prepared to handle an invalid one, so I think we should avoid giving this type an invalid value in the first place.